src/kit.ts

```diff
@@ -734,8 +734,10 @@ async function collectDevBatVars(hostArch: string, devbat: string, args: string[
     const existPath = vars.get('PATH') ?? '';
     const oldWinSdkBinPath = path.join(WindowsSdkDir, 'bin', hostArch);
     const newWinSdkBinPath = path.join(WindowsSdkDir, 'bin', WindowsSDKVersion, hostArch);
-    if (existPath.toLowerCase().indexOf(oldWinSdkBinPath.toLowerCase()) >= 0
-      && existPath.toLowerCase().indexOf(newWinSdkBinPath.toLowerCase()) < 0) {
+    const newWinSdkBinPathExist = await fs.exists(newWinSdkBinPath);
+    if (newWinSdkBinPathExist &&
+        existPath !== '' &&
```

Comment:
I wouldn't think this check is needed.  It should fail the next check without causing a crash.

Reply:
It's needed, as https://github.com/microsoft/vscode-cmake-tools/pull/1763 are didn't merged yet
On Windows 10, the ‘PATH' turn out to be 'Path', so existPath can be empty, when existPath is '', we can not override with ’PATH'

Reply:
Ah. You're right. I misread the intent.